src/server/scripts/Commands/cs_ticket.cpp

```diff
@@ -199,7 +199,7 @@ class ticket_commandscript : public CommandScript
         ticket->SaveToDB(trans);
         sTicketMgr->UpdateLastChange();
 
-        std::string msg = ticket->FormatMessageString(*handler, NULL, ticket->GetAssignedToName().c_str(), NULL, NULL, NULL);
+        std::string msg = ticket->FormatMessageString(*handler, NULL, NULL, NULL, NULL, NULL);
```

Comment:
What's the point of his line after this is removed? It's not passing any data into the function at all, it's all NULLs